docs/usage/rclone/index.rst

```diff
@@ -5,4 +5,189 @@ Rclone-based replication
 .. toctree::
    :hidden:
 
-Work in progress
+   database_example
+
+.. sidebar:: Contents
+
+   .. contents:: Rclone-based replication
+
+Rclone-based replication supports 1:1 asynchronous replication of volumes for use
+cases such as:
+
+- High fan-out data replication from a central site to many (edge) sites
+
+With this method, Scribe synchronizes data from a ReplicationSource to a ReplicationDestination
+using `Rclone <https://rclone.org/>`_ using an intermediary storage system like AWS S3. 
+
+----------------------------------
+
+The Rclone method uses a "push" and "pull" model for the data replication. A schedule or other 
+trigger is used on the source side of the relationship to trigger each replication iteration.
+
+Following are the sequences of events happening in each iterations.
+
+- A point-in-time (PiT) copy of the source volume is created using CSI drivers. It will be used as the source data .
+- A temporary PVC is created out of the PiT copy and mounted on Rclone data mover job pod.
+- The Scribe Rclone data mover then connects to the intermediary storage system (e.g. AWS S3) using configurations
+  based on ``rclone-secret``. It uses ``rclone sync`` to copy source data to S3.
+- At the conclusion of the transfer, the destination creates a PiT copy to preserve the incoming source data.
+
+Scribe is configured via two CustomResources (CRs), one on the source side and
+one on the destination side of the replication relationship.
+
+Source configuration
+=========================
+
+Start by configuring the source; a minimal example is shown below:
+
+.. code:: yaml
+
+   ---
+   apiVersion: scribe/v1alpha1
+   kind: ReplicationSource
+   metadata:
+     name: database-source
+     namespace: source
+   spec:
+    sourcePVC: mysql-pv-claim
+    trigger:
+      schedule: "*/6 * * * *"
+     rclone:
+       copyMethod: Snapshot
+       capacity: 10Gi
+       accessModes: ["ReadWriteOnce"]
+
+Since the ``copyMethod`` specified above is ``Snapshot``, the Rclone data mover creates a ``VolumeSnapshot`` 
+of the source pvc ``mysql-pv-claim`` using the ``VolumeSnapshotClass`` provisioned by the CSI driver.
+
+It then creates a temproray pvc ``scribe-src-database-source`` out of the VolumeSnapshot to transfer source
+data to the intermediary storage system like AWS S3 using the configurations provided in ``rclone-secret``
+
```

Comment:
How about adding some more information on the `rclone-secret` here (i.e., that it's just an rclone config file)?
A copy of the rclone.conf plus the command to load it as a secret `kubectl create secret generic ...blah...` would be good

Reply:
The command to create the secret is included in database_example.rst. I will explain the contents and probably include the link to Rclone AWS docs.